AmplifyTestCommon/Models/CustomPrimaryKey/primarykey_schema.graphql

```diff
@@ -53,3 +45,26 @@ type CommentWithCompositeKey @model {
   content: String!
   post: PostWithCompositeKey @belongsTo
 }
+
+type PostWithTagsCompositeKey @model {
```

Comment:
can we add a comment to section off between groups of types that are used together? hard to tell which types are together. the comment can as simple as just the types or description of the relationship we're testing, ie
```
# Composite Key Many-to-Many (Post has-many Tag, Tag has-many Post)
```